common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -328,7 +201,7 @@ def new_lcp(self, state, text=None):
             text = self.data
 
         capa_system = LoncapaSystem(
-            ajax_url=self.runtime.ajax_url,
+            ajax_url=self.ajax_url,
```

Comment:
The one in `ModuleSystem` works only for `XModules` so added one in `XModuleToXBlockMixin`.